### ISSUE TITLE
Hide empty columns in table

### ISSUE THREAD

**Issue by timwis** [timwis/vizwit#105]

**Comment by marks** [comment]
My personal opinion is that empty data is data. I think #123 takes care of this.. or maybe there is a config flag to opt into this. Just my 2c

**Comment by timwis** [comment]
I agree @marks -- this is actually a socrata-provider-specific issue: the fields/metadata endpoint gives back a bunch of those "magic" fields (the computed polygon ones, and like 5 location ones). There's not a clear way to filter them out when fetching the fields, so the table view renders them as columns. But when you get the actual data/rows, those fields don't exist, so they show up as empty columns. So I'll reword this issue to reference those magic fields (let me know if there's a more technical name for them)

**Comment by marks** [comment]
I believe we call these computed fields. Right now they only ever seem to have the format of  `:@computed_region_a1b2_c3d4` but I think it is safe to assume they will always start with `:@` where `:` denotes a special column and `@` denotes it is computed.

**Comment by timwis** [comment]
@marks good idea - it would be in socrata-fields.js, which is currently being refactored in #149.

The only other thing to look at is that socrata adds like 5 "location" fields too, and those seem to be blank as well. (location zip, location state, etc.)
